packages/agent/src/browser.ts

```diff
@@ -31,8 +31,8 @@ export const getName = (d: Document) => {
 }
 
 export const getOffset = (w: Window) => ({
-  x: w.scrollX || w.pageXOffset,
-  y: w.scrollY || w.pageYOffset
+  left: w.scrollX || w.pageXOffset,
```

Comment:
why renamed?
I guess, `left` is using to express position in window, and `x` is using to window position.

Reply:
check here too.
https://github.com/userdive/agent.js/pull/936/files#diff-d1a34aee5de190386e79152f0619dcd9R104

Reply:
This for use same namespace.

https://github.com/userdive/agent.js/pull/936/files#diff-e354d7d69dd5b38778b6bbe49c0a939eL82

Reply:
Namespace

```
x: pageX
y: pageY
left: scrollX
top: scrollY
```